pkgs/applications/science/biology/samtools/default.nix

```diff
@@ -38,7 +38,8 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  doCheck = true;
+  # tests require `bgzip` from the htslib package
```

Comment:
the proper fix is to move or add htslib to `nativeBuildInputs`

Reply:
I was going to suggest that as well but it seems like `htslib` is needed both for that binary and as an actual library. If it was added to both, would things still function as expected?

Reply:
> would things still function as expected?

yes because strictDeps

Reply:
strictDeps isn't default (as of now), so that would have to be set as well, right?

Reply:
it's on when cross-compiling, thats why bgzip isn't in $PATH